### ISSUE TITLE
Accessibility Feedback Channel

### ISSUE THREAD

**Issue by sana-ajani** [microsoft/vscode#93499]
Hi there!

One of our team's top priorities is making Visual Studio Code a more accessible product and improving experiences for every user. We would like to create an accessibility community, where VS Code users can directly give our team feedback, voice problems, and share accessibility practices.  

Today, many people use GitHub to open issues regarding accessibility and issues they run into using VS Code with various assistive technologies, but we realize there's more voices and people that are not on GitHub or want to share their experience through another channel.

We are open to suggestions for the best way to regularly talk to our users. Some ideas we have are: 

- @isidorn and I hold a monthly accessibility call, where users can bring up issues they've experienced in parts of the editor. We can post an issue about the call each month on GitHub and tag our most active users, share on Twitter, and post about it on other accessibility or a11y communities.
- Create a specific Slack community (or other medium) that anyone who uses VS Code with assistive technologies can join and regularly post issues or bugs they are running into

From these channels, we can create individual GitHub issues and track the progress of the work there.

### Feedback
This is how we are thinking of moving things forward, but we are very open to feedback as to what the community thinks and needs. So please chime in and leave your thoughts and feedback.

Thank you!

**Comment by isidorn** [comment]
fyi @zersiax @jvesouza @ajborka @neurrone @redeyedragon @joanmarie @pawelurbanski  @mehgcap @webczat @ndarilek @alex19EP @MarcoZehe @leonardder

**Comment by RedEyeDragon** [comment]
@sana-ajani @isidorn 

This is a great idea. I for one live on Slack and think a Slack channel would be perfect for this type of feedback/discussion. In addition it would be great to have time via voice/video chat periodically. 

Can I just say I appreciate all the work that has gone into VS Code thus far. I have been using VS Code exclusively since @isidorn introduced me to the insiders build and I love it more and more everyday! Thank you!

**Comment by MarcoZehe** [comment]
Slack would work for me, too. But I am also at home on Github, so don't mind either way.

**Comment by pawelurbanski** [comment]
VS Code has reached very good maturity/stability level to be a very practical IDE. While there will be something to fix here and there every so often, it should start gaining more users in terms of absolute numbers.
When it comes to usage/popularity and accessibility support we have a kind of chicken and egg problem. You need something convenient to use when you are getting started with some tech-related work, unless you have it, it is hard to get started for you did not get the chance to develop enough skills to fix it for example or find work arounds – a bit of a vicious circle.
I am slowly working on a regulr document that addresses not only accessibility-related ideas, but also very likely good use cases when it comes to technical roles that are the most friendly to those who happen to be disabled. Tools are just one part of the story – tapping into regular processes, team-work, communication is a wider, but very important scope.

**Comment by alex19EP** [comment]
yep. i think slack room is good idea

**Comment by mehgcap** [comment]
I've only used Slack briefly, a couple years ago. I can give it another try, though.

--
Alex Hall

**Comment by webczat** [comment]
tbh, let's be consistent. I believe that vscode has a gitter channel, right? I don't know how gitter works, can another room be created for accessibility of vscode without creating a repo?

**Comment by ajborka** [comment]
@sana-ajani  Slack channel should work. At least the last time I tried Slack some years ago it worked.

**Comment by leonardder** [comment]
This is a great idea.
Honestly, I don't consider Slack one of the most accessible solutions out there. I'm also happy with Github. May be just a specific repo for the a11y docs in which we can provide issues? There are more repositories used like this, see for example https://github.com/FreedomScientific/VFO-standards-support

**Comment by jvesouza** [comment]
The idea is certainly excellent, but I'm just not sure that slack is the best way from the point of view of accessibility.
The last time I tried slack I wasn't very happy, but I have to admit that I didn't make a big investment and that I tried it for a long time. Things must certainly have changed for the better.

**Comment by ndarilek** [comment]
I, too, have reservations about Slack. It seems heavyweight for feedback 
on a specific aspect of a specific tool. If there was an entire VSCode 
Slack then I'd understand an accessibility channel being created. Also, 
I've never really been happy with its accessibility. Seems to have 
gotten better, but it doesn't even use live regions for its chat area, 
which is such an easy access win that I have a hard time thinking they 
take us seriously without it.


Gitter seems like the simplest, most accessible channel. Otherwise, I'm 
partial to [Matrix](https://matrix.org).

**Comment by zersiax** [comment]
Hmm. Last time I checked, Gitter wasn't the poster child for screen reader accessibility either, has this improved?
Slack is certainly usable, the lack of messages being automatically read as they come in can admittedly be an obstacle though, mostly because people will inadverntently "chat over each other" as it were.

**Comment by ndarilek** [comment]
I certainly haven't done an audit, but it's at least easy to get in, 
chat, hear incoming messages, etc. I don't even understand how to use 
Slack threads, and sometimes I get replies in a thread while others 
messages just go into the chat. I don't imagine this stuff is difficult 
to figure out--I just usually get slammed with having to use Slack for 
some obscure project or other, not daily for work.

**Comment by leonardder** [comment]
As VSCode is a Microsoft product, shouldn't we discuss the use of Teams 
instead of SLack😜?

**Comment by ajborka** [comment]
@leonardder  teams might not work for everyone. Last I knew, teams didn't work with OSX or Linux distros because you had to use a native Windows 10 app+server install to make it work.

**Comment by webczat** [comment]
Teams has a web app too, and I believe there is also a linux version now, but haven't tested if it's accessible. That said, not sure if that's appropriate to use it or not. Gitter seems more inline with some current practices, like dotnet repos.

**Comment by ndarilek** [comment]
Teams installed via Flatpak doesn't appear to be accessible--at least, 
passing in `--force-renderer-accessibility` gets me nothing.


If we need something other than issues, let's just go with Gitter. 
Everyone here already has a GitHub account, and I don't really want to 
create yet another Slack or Teams account. And as much as I like Matrix, 
that same justification prevents me from advocating for it too much 
since I imagine folks not on Matrix don't want that account either 
(though you really should check it out, as it is now my cross-device IRC 
bouncer among other things. :)

**Comment by webczat** [comment]
I am literally using teams as a webapp, works well and is accessible, at least reasonably, although I'd say that sometimes it's non obvious.

**Comment by sana-ajani** [comment]
Hey everyone, thank you for the feedback. Based on the comments, we've created a Gitter channel: https://gitter.im/Microsoft/vscode-a11y 

This will serve as an ongoing feedback channel, where you can chat directly with us and other users, post issues you're running into, and where we will coordinate details for a call.

**Comment by isidorn** [comment]
@sana-ajani thanks for doing this.
Thus we are clossing this issue.
